src/main/java/ev3dev/hardware/EV3DevPlatforms.java

```diff
@@ -28,7 +30,65 @@ public EV3DevPlatform getPlatform() {
     }
 
     public String getMotorPort(final Port port) {
-        return instance.getMotorPort(port);
+
+        //TODO Pending to review in detail
+        //return instance.getMotorPort(port);
+
```

Comment:
This is probably a wrong place to do such a revert. If you want to use the original implementation, switch it in EV3DevPlatformsImpl. Changing it there defeats the caching.